wasmcloud-test-util/src/lib.rs

```diff
@@ -64,11 +72,11 @@ macro_rules! run_selected {
             let re = match $crate::regex::Regex::new(pattern) {
                 Ok(re) => re,
                 Err(e) => {
-                    let error = wasmbus_rpc::RpcError::Other(format!(
+                    let error = RpcError::Other(format!(
                         "invalid regex spec '{}' for test case: {}",
                         pattern, e
                     ));
-                    results.push(("TestCase", Err::<(),wasmbus_rpc::RpcError>(error)).into());
+                    results.push(("TestCase", Err::<(),RpcError>(error)).into());
```

Comment:
Wasn't there an issue with not specifying fully qualified struct paths? I believe @thomastaylor312 had a problem with stderr but it might be better to keep this as `wasmbus_rpc::RpcError` pending his reply

Reply:
the problem Taylor ran into was that `Result` had not been specified. In an earlier PR I changed the macro to emit std::result::Result instead of `Result`.
As for RpcError, it's not standard, and we define the path so there's no conflict. Also, the path changed from wasmbus_rpc::RpcError to wasmbus_rpc::error::RpcError.